File: Depreciated Java Version/One Time Pad Library/src/VernamKeyGenerator.java
import java.security.SecureRandom;
import java.util.BitSet;

/**
 * Created by Oliver on 17/05/2018.
 * Written by Oliver Bathurst <oliverbathurst12345@gmail.com>
 */

class VernamKeyGenerator {
    private final SecureRandom secureRandom = new SecureRandom();

    VernamKeyGenerator(){}

    VernamKey generateKey(int length){
        byte[] keyBytes = new byte[length];
        secureRandom.nextBytes(keyBytes);
        return new VernamKey(BitSet.valueOf(keyBytes));
    }
}

File: CSharp version/One Time Pad/One Time Pad CS/VernamKeyGenerator.cs
using System.Security.Cryptography;
using System.Collections;

class VernamKeyGenerator{
    private RandomNumberGenerator secureRandom = RandomNumberGenerator.Create();

    public VernamKeyGenerator() {}

    public VernamKey generateKey(int length){
        byte[] keyBytes = new byte[length];
        secureRandom.GetBytes(keyBytes);
        return new VernamKey(new BitArray(keyBytes));
    }
}